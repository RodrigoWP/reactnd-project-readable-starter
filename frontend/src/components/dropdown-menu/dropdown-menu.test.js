import React from 'react'
import { shallow } from 'enzyme'
import { DropdownMenu } from './index'

const { DropdownItem } = DropdownMenu

it(' Dropdown menu should render correctly', () => {
  const component = shallow(<DropdownMenu />)
  expect(component).toMatchSnapshot()
})

it('Dropdown item should render correctly', () => {
  const component = shallow(<DropdownItem />)
  expect(component).toMatchSnapshot()
})
