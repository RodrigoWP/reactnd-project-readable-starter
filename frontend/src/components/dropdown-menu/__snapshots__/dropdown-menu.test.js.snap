// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Dropdown menu should render correctly 1`] = `
<div>
  <img
    src={Object {}}
  />
  <div />
</div>
`;

exports[`Dropdown item should render correctly 1`] = `
<div>
  <span>
    Editar
  </span>
</div>
`;
